### DISCUSSION TITLE
Help building node-libtorrent

### DISCUSSION THREAD

**Discussion by RProduction** [arvidn/libtorrent#7507] [Q&A]
Hello, I need help building binding for node. I already followed the instructions and successfully built the libtorrent using b2. It results into lib file and obj files.
How can I use it as a base for building binding for node?
When I try npm install node-libtorrent it fail to build because it probably need the prebuilt libtorrent file.

Can you give me some insight how to build the binding?

**Comment by arvidn** [comment]
you would be best off asking whoever wrote the node bindings.

> When I try npm install node-libtorrent it fail to build because it probably need the prebuilt libtorrent file.

The error message you get when trying to build might tell you for sure

**Comment by RProduction** [comment]
Oh no... I think binding it's official binding for this library. The node-libtorrent binding repository seem like not have activities since a few years ago.

Edit:
I am decided to change the library for torrenting into webtorrent. Seem it can be use in node environment and also support normal tcp/udp protocol.
Ah, it is really tought experience building library from scratch. I don't have any experience in c++ field lol.
